docs/references/studio/api/jobs.md

```diff
@@ -0,0 +1,358 @@
+- `credentials_name` (body, optional): Credentials identifier
+- `team_name` (body, required): Team identifier (automatically added by client)
+
+Check documentation from [`datachain job run`](../../../commands/job/run.md) for more information about the parameters above.
+You can get the file id from above (Upload file) endpoint.
+
+**Example Request**:
+```bash
+curl -X POST "https://studio.datachain.ai/api/datachain/job" \
+  -H "Authorization: token YOUR_STUDIO_TOKEN" \
+  -H "Content-Type: application/json" \
+  -d '{
+        "query": "print(1)",
+        "query_type": "PYTHON",
+        "environment": "ENV_FROM_FILE=1\nENV_FROM_ARGS=1",
+        "workers": 2,
+        "query_name": "example_query.py",
+        "files": ["1"],
+        "python_version": "3.12",
+        "requirements": "pyjokes\nstupidity",
+        "team_name": "team_name",
+        "repository": "https://github.com/iterative/datachain",
+        "priority": 5,
+        "compute_cluster_name": "default",
+        "start_after": null,
+        "cron_expression": null,
+        "credentials_name": "my-credentials",
+    }'
+```
+
+**Example Response**:
+```json
+{
+    "job": {
+        "id": "a184a035-793a-4d04-8ba5-a2153fe72182",
+        "url": "https://studio.datachain.ai/team/Iterative/datasets/jobs/a184a035-793a-4d04-8ba5-a2153fe72182"
+    }
+}
+```
+
+**Using Studio Client**:
+```python
+from datachain.remote.studio import StudioClient
+
+client = StudioClient(team="my-team")
+response = client.create_job(
+    query="print(1)",
+    query_type="PYTHON",
+    workers=2,
+    query_name="Data Analysis Job",
+    priority=3
+)
+
+if response.ok:
+    job_id = response.data['id']
+    print(f"Job created with ID: {job_id}")
+```
+
+---
+
+### List Jobs
+
+**Description**: Retrieve a list of jobs with optional status filtering.
+
+**Endpoint**: `/datachain/jobs`
+
+**Method**: `GET`
+
+**Authentication**: Requires `read` access to `JOBS` scope from [`datachain auth login`](../../../commands/auth/login.md)
+
+**Parameters**:
+
+- `status` (query, optional): Filter by job status
+- `limit` (query, optional): Maximum number of jobs to return (default: 20)
+- `team_name` (query, required): Team identifier (automatically added by client)
+
+**Example Request**:
+```bash
+curl -X GET "https://studio.datachain.ai/api/datachain/jobs?status=running&limit=10&team_name=my-team" \
+  -H "Authorization: token YOUR_STUDIO_TOKEN" \
+  -H "Content-Type: application/json"
+```
+
+**Example Response**:
+```json
+{
+  "jobs": [
+    {
+      "created_at": "2025-08-18T09:31:11.896956+00:00",
+      "created_by": "amritghimire",
+      "environment": {},
+      "error_message": "",
+      "error_stack": "",
+      "exit_code": null,
+      "finished_at": null,
+      "id": "a184a035-793a-4d04-8ba5-a2153fe72182",
+      "name": "test.py",
+      "python_version": null,
+      "query": "",
+      "query_type": 1,
+      "repository": "",
+      "requirements": "",
+      "status": "CREATED",
```

Comment:
Here `status` is a string. I believe we should be consistent within the whole API.